### PR TITLE
Support for dragging outside the widget bounds

### PR DESCRIPTION
**Pull request by keianhzo**
When you are press and drag outside the widget bounds the input stops. This adds support for dragging outside the widget bounds. I think it is a more natural behavior for the input.

I started this branch a while ago so I guess it's a good moment to share and check if this is something that we like.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[*]
```
Comment (MortimerGoro) [inline .editorconfig]: If we still need this file, can we separate it to a different PR?

**Comment by keianhzo** [comment]
@MortimerGoro updated

**Comment by bluemarvin** [comment]
@keianhzo Looks like this conflicts with the two controller typing.

**Comment by bluemarvin** [comment]
I got this crash when trying to resize the window https://crash-stats.mozilla.com/report/index/6e253a60-c37d-4cdf-8954-0d9940200403

**Comment by bluemarvin** [comment]
Crash is here: https://github.com/MozillaReality/FirefoxReality/pull/2745/files#diff-96af3977b6728c8ea9f0abae3d641614R421
`widget` is `null`

**Comment by bluemarvin** [comment]
I'm still seeing the crash.

**Comment by keianhzo** [comment]
@bluemarvin I've pushed a fix for the resize issue
